src/components/Stream.tsx

```diff
@@ -267,11 +267,11 @@ export const Stream = ({ className = '' }) => {
       }
       engineCommandManager.sendSceneCommand(command).then(async () => {
         if (!context.sketchPathToNode) return
-        const varDec = getNodeFromPath<VariableDeclarator>(
+        getNodeFromPath<VariableDeclarator>(
```

Comment:
Also curious about those. To get the linter happy I removed the var declaration, but do we actually need `getNodeFromPath` executed for something?

Reply:
@Irev-Dev